### DISCUSSION TITLE
bito Language

### DISCUSSION THREAD

**Discussion by wuwei0727** [gitbito/bitoai#216] [Q&A]
![image](https://github.com/gitbito/bitoai/assets/100761249/a81b9a84-9efe-4fa6-949b-44c0d7d8419d)
What is the use of this setting? I used IDEA and it did not reply according to the language I set.
![image](https://github.com/gitbito/bitoai/assets/100761249/296f8243-6045-4ffe-b7f3-2c24646d20f1)

**Comment by nisha-bito** [comment]
hi @wuwei0727  - 

If you ask question in English, it will respond back in the English. If you type your question in Arabic, it will respond back in Arabic.   
If you want to ask question in English, please append " reply in Arabic" in your question/prompt to double sure to get the answer in Arabic. 

thanks,
Nisha

**Reply by wuwei0727** [reply]
![image](https://github.com/gitbito/bitoai/assets/100761249/d1dcc124-8c97-4f6a-97c6-eb27131887c4)
Hello, can you consider setting a language selection in jetbrains IDEA? The output will be in whatever language I set, regardless of any language I input.
